config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -518,16 +518,19 @@ presubmits:
           value: "1"
 
     # This is the default e2e test for all PRs.
-  - name: pull-cert-manager-e2e-v1-23
+  - &default-e2e
+    name: pull-cert-manager-e2e-v1-23
     context: pull-cert-manager-e2e-v1-23
-    always_run: true
-    optional: false
+    always_run: false
+    optional: true
     max_concurrency: 4
     agent: kubernetes
     decorate: true
     branches:
     - master
-    - release-1.7
+    - release-.*
+    skip_branches:
+    - release-1.5
```

Comment:
Oops, `branches` and `skip_branches` are mutually exclusive: https://github.com/kubernetes/test-infra/blob/151f4dbb9106a20a901ae7b350c4fb3b1e4ee17c/prow/config/jobs_test.go#L93